openff/bespokefit/tests/executor/services/fragmenter/test_worker.py

```diff
@@ -65,3 +71,42 @@ def test_fragmentation_skip(bace):
     )
 
     assert result_json == "null"
+
+
+def test_fragmentation_symmetry_fragments():
+    """Make sure symmetry equivalent fragments are filtered"""
+
+    # molecule has symmetry equivalent torsions
+    molecule = Molecule.from_smiles("COP(=O)(O)OC")
+
+    result_json = worker.fragment(
+        cmiles=molecule.to_smiles(mapped=True),
+        fragmenter_json=PfizerFragmenter().json(),
+        target_bond_smarts=[_DEFAULT_ROTATABLE_SMIRKS],
+    )
+    result = FragmentationResult.parse_raw(result_json)
+
+    assert len(result.fragments) == 1
+    assert len(result.fragment_molecules) == 1
+
+
+def test_fragmentation_equivalent_no_symmetry():
+    """Make sure duplicated fragments which are not symmetry equivalent are not filtered."""
+
+    molecule = Molecule.from_file(
+        file_path=get_data_file_path(
+            os.path.join("test", "molecules", "tyk2", "ejm31.sdf"), "openff.bespokefit"
+        ),
+        file_format="sdf",
+    )
```

Comment:
Is there perhaps a smaller / cheaper molecule which should test this behaviour so we don't have to fragment a larger molecule everytime and keep the unit tests as quick as possible? +1 if can also add a check that the output de-duplicated fragments match a set of expected ones!

Reply:
Good spot yeah I think that is what the bace files were originally for I have updated the test to use these which is a slightly smaller molecule and still covers what I wanted to test!